packages/lodestar/src/network/gossip/validator.ts

```diff
@@ -69,17 +69,17 @@ export class GossipMessageValidator implements IGossipMessageValidator {
         case BlockErrorCode.ERR_PROPOSAL_SIGNATURE_INVALID:
         case BlockErrorCode.ERR_INCORRECT_PROPOSER:
         case BlockErrorCode.ERR_KNOWN_BAD_BLOCK:
-          this.logger.warn("Rejecting gossip block", {...e.getMetadata(), ...logContext});
+          this.logger.warn("Rejecting gossip block", toJson({...e.getMetadata(), ...logContext}));
```

Comment:
You shouldn't have to call `toJson` on the logger context. It is already called here
https://github.com/ChainSafe/lodestar/blob/10e261546c4c054811e272ebcb12bf736ef627dd/packages/lodestar-utils/src/logger/format.ts#L84
Why is it necessary? Doing 
```ts
this.logger.warn("Rejecting gossip block", {...e.getMetadata(), ...logContext});
```
results in bad logs?

Reply:
Yeah, in the case of an error having some metadata that is not immediately printable (such as a Uint8Array), it doesn't print it correctly and must be converted to hex string first.

Reply:
I was trying to call it here before I found out that toJson had a bug, so maybe I don't need to call it anymore.

Reply:
Yup, don't need to call it.  Just pushed up a change for it.